## Introduction
I am a principal investigator in a [genomics lab](http://genomics.usc.edu) at the University of Southern California. Lab members and collaborators developed a number of bioinformatics software tools for analysis of genomics data sets. Some of these tools are available at github repositories:

- [SeqMule](https://github.com/WangGenomicsLab/SeqMule): Automated human exome/genome variants detection
- [iCAGES](https://github.com/WangGenomicsLab/icages): Prioritizing cancer driver genes and predicting drugs from sequencing data
- [iCAGES-server](https://github.com/WangGenomicsLab/icages-server): a web interface to iCAGES running on Ruby on Rails
- [Phenolyzer](https://github.com/WangGenomicsLab/phenolyzer): phenotype-based prioritization of candidate genes for human diseases
- [lncScore](https://github.com/WangGenomicsLab/lncscore): Prediction of lncRNA from RNA-Seq data
- [PennCNV](https://github.com/WangGenomicsLab/penncnv): Copy number variation detection from SNP arrays (written in Perl+C)
- PennCNV2: Copy number variation detection from sequencing data