## Introduction
I am a principal investigator in a [genomics lab](http://genomics.usc.edu) at the University of Southern California. Lab members and collaborators developed a number of bioinformatics software tools for analysis of genomics data sets. Some of these tools are available at github repositories:

- [SeqMule](https://github.com/WGLab/SeqMule): Automated human exome/genome variants detection
- [iCAGES](https://github.com/WGLab/icages): Prioritizing cancer driver genes and predicting drugs from sequencing data
- [iCAGES-server](https://github.com/WGLab/icages-server): a web interface to iCAGES running on Ruby on Rails
- [Phenolyzer](https://github.com/WGLab/phenolyzer): phenotype-based prioritization of candidate genes for human diseases
- [lncScore](https://github.com/WGLab/lncscore): Prediction of lncRNA from RNA-Seq data
- [PennCNV](https://github.com/WGLab/penncnv): Copy number variation detection from SNP arrays (mixture of Perl and C)
- [PennCNV-Tumor](https://github.com/WGLab/PennCNV2): Rapid copy number variation and copy number alteration detection from SNP arrays (C++)
- [PennCNV-Hadoop](https://github.com/WGLab/PennCNV3): Hadoop-based copy number variation detection from sequencing data (Java)
- [doc-ANNOVAR](https://github.com/WGLab/doc-ANNOVAR): documentation for the ANNOVAR software (source code not available at github due to institutional restrictions)
- [GenGen](https://github.com/WGLab/GenGen): A set of software tools to facilitate GWAS analysis
- [UROBORUS](https://github.com/WGLab/UROBORUS): Circular RNA identification from RNA-Seq data
